packages/nextlove/src/generators/lib/zod-openapi.ts

```diff
@@ -495,9 +495,17 @@ function parseUnion({
     }
   }
 
+  const isNullable = contents.some(
+    (content) => content._def.typeName === "ZodNull"
```

Comment:
I wonder if we'd need to unwrap these types, e.g. what does `z.string().nullable().optional()` product - is it a `ZodNull` or `ZodOptional` that contains a `ZodNull`

Reply:
It think it'll be unwrapped further on line 507 by `generateSchema`. This change covers only `z.null()` so that `z.union([z.string(), z.boolean(), z.null()])` becomes:
```
my_property: {
  oneOf: [
    {
      type: "string",
    },
    {
      type: "boolean",
    },
  ],
  nullable: true,
}
```

and not:

```
my_property: {
  oneOf: [
    {
      type: "string",
    },
    {
      type: "boolean",
    },
    {
      type: "null",
    },
  ],
}
```